automates/text_reading/src/main/scala/org/clulab/aske/automates/actions/ExpansionHandler.scala

```diff
@@ -560,11 +570,67 @@ object ExpansionHandler {
     "acl:relcl".r,
     "^nmod_for".r,
     "nmod_at".r,
-//    "^nmod_of".r,
+    "^nmod_of".r,
     "nmod_under".r,
 //    "nmod_in".r
 //    "aux".r
   )
 
+  val INVALID_OUTGOING_MODELDESCR = Set[scala.util.matching.Regex](
+    "acl:relcl".r,
+    "acl_until".r,
+    "advcl_to".r,
+    "advcl_if".r,
+    "^advcl_because".r,
+//    "advmod".r,
+    "^case".r,
+    "^cc$".r,
+    "ccomp".r,
+    //    "compound".r,
+//    "^conj".r,
+    "cop".r,
+    "dep".r, //todo: expansion on dep is freq too broad; check which tests fail if dep is included as invalid outgoing,
+    "nmod_at".r,
+    "nmod_through".r,
+    "^nmod_as".r,
+    "^nmod_because".r,
+    "^nmod_due_to".r,
+    "^nmod_except".r,
+    "^nmod_given".r,
+    "^nmod_since".r,
+    "^nmod_without$".r,
+//    "nmod_in".r,
+    //    "nmod_by".r,
+    //    "nummod".r,
+//    "^nsubj".r,
+    "^punct".r,
+    "^ref$".r,
+    "appos".r,
```

Comment:
do you think it's safe to remove the commented out relations? just to clean up?

Reply:
Yes, I don't see any harm. I'll clean up those in this branch. Also, I've noticed some disabled lines in ExpansionHandler - would it be okay to clean-up those too? (see below for examples)

`//    println(s"$proportion --> ${mention.sentenceObj.getSentenceText}"`
`//          println("expanded arg: " + expandedWithIdentifierLabelDropped.text + " " + expandedWithIdentifierLabelDropped.foundBy + " " + expandedWithIdentifierLabelDropped.label + "|" + expandedWithIdentifierLabelDropped.labels.mkString("::"))`
`          //.map(addSubsumedAttachments(_, state))
          //.map(attachDCT(_, state))
          //.map(addOverlappingAttachmentsTextBounds(_, state))`
`//println(s"orig: ${orig.text}\texpanded: ${expanded.text}")`
`//    println("\noriginal:  " + entity.text + " " + entity.labels.mkString("") + " " + entity.foundBy)
//    println("expanded:  " + incomingExpanded.text + " " + incomingExpanded.labels.mkString("") ++ incomingExpanded.foundBy)` and a few more like these (+ is "attachments helper methods" on line 385 needed here?)

Reply:
keep printouts---I use them for debugging. yesterday I thought I'd had to write printouts again but luckily they were already there :D

Reply:
Okay, and "attachments helper methods" are also needed there?

Reply:
yeah you can get rid of those helper methods. our attachments are different than what they do in the project that was used as a template for our system.